docs/reference/filters.md

```diff
@@ -107,11 +107,15 @@ The available operators are:
 * `lt` / `lte` for 'lower than' / 'lower than or equal',
 * `nil` for 'empty', and `not_nil` for non-empty (any value),
   * Note that `nil` and `not_nil` are the only operators that do not require anything inside `values`.
+* `starts_with` / `not_starts_with` / `ends_with` / `not_ends_with` / `contains` / `not contains` are available for searching in short string fields (name, value, title...).
+* `search`  is available in short string and text fields.
 
 When using a numerical comparison operators (`gt` and the like) against textual values, the alphabetical ordering is used.
```

Comment:
maybe write this precision just after the 'gt' line

Reply:
Not sure about understanding this point ?